apps/web/app/api/organizations/[org]/subteams/route.ts

```diff
@@ -0,0 +1,38 @@
+import type { Params } from "app/_types";
```

Comment:
For easier review,

Left: App Router

Right: Pages Router

<img width="1469" alt="Screenshot 2025-03-11 at 11 48 12 PM" src="https://github.com/user-attachments/assets/856d9b1d-cc66-4520-92ff-2ec226170d2d" />